elements/lisk-transactions/src/register_multisignature_account.ts

```diff
@@ -95,34 +120,45 @@ const validateInputs = ({
 export const registerMultisignature = (
 	inputs: RegisterMultisignatureInputs,
 ): Partial<TransactionJSON> => {
-	validateInputs(inputs);
 	const {
-		keysgroup,
-		lifetime,
-		minimum,
-		passphrase,
+		senderPassphrase,
+		passphrases,
+		mandatoryKeys,
+		optionalKeys,
+		numberOfSignatures,
 		networkIdentifier,
+		fee,
+		nonce,
 	} = inputs;
 
-	const plusPrependedKeysgroup = prependPlusToPublicKeys(keysgroup);
+	validateInputs({
+		mandatoryPublicKeys: mandatoryKeys,
+		optionalPublicKeys: optionalKeys,
+		numberOfSignatures,
+		networkIdentifier,
+		fee,
+		nonce,
+	});
 
 	const transaction = {
 		...createBaseTransaction(inputs),
 		type: 12,
+		networkIdentifier,
 		asset: {
-			min: minimum,
-			lifetime,
-			keysgroup: plusPrependedKeysgroup,
+			mandatoryKeys,
+			optionalKeys,
+			numberOfSignatures,
 		},
-		networkIdentifier,
 	};
 
```

Comment:
for now, maybe keep this check `if(!passphrases.length)` so that it can create non-signed transaction (it's same for other transaction creation function)
We can decide if we want to keep this feature or not, and remove this all at once if not needed